project/AppDependencies.scala

```diff
@@ -22,18 +22,19 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"   %% "bootstrap-backend-play-28" % "5.3.0",
-    "org.typelevel" %% "cats-core"                 % "2.6.1",
-    "com.chuusai"   %% "shapeless"                 % "2.4.0-M1"
+    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % "5.20.0",
```

Comment:
When "bootstrap-backend-play-28" is updated to version 5.18+ you now have to amend the root level logger. This is the documentation for it.https://github.com/hmrc/bootstrap-play#version-5180  Similar to [https://github.com/hmrc/goods-movement-system-frontend/commit/ca7031c9417dfc83c3992b7d5f3742fd220976fa#diff-d93ae00fd098f93a[…]20e94bd374fbb797afec6](https://github.com/hmrc/goods-movement-system-frontend/commit/ca7031c9417dfc83c3992b7d5f3742fd220976fa#diff-d93ae00fd098f93a6584bd8abafba9f5d9a3b12364d20e94bd374fbb797afec6) but you also need to amend it in app-config-production like this https://github.com/hmrc/app-config-production/pull/13545/files